onnx/backend/test/case/node/cast.py

```diff
@@ -17,6 +17,8 @@
     tensor_dtype_to_field,
 )
 from onnx.numpy_helper import float8e4m3_to_float32, float8e5m2_to_float32
+from onnx.reference.custom_element_types import int4, uint4
```

Comment:
I would import modules only for any new code: https://gist.github.com/justinchuby/9085242a53158f2fd7ae7aa650e55ee3#3-import-only-modules

-----------

I would avoid importing a class or a function from a module.

Import only the module instead to keep the name space clean and provide readers with more context on where its members come from. For example, readers don’t need to backtrack to see `sleep` is a function from `time`, as opposed to a function defined in the file. https://google.github.io/styleguide/pyguide.html#22-imports